scripts/cluster

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Reset test-dirs
+rm -rf test-chain-*
+rm -f genesis.json
+
+go build -o polygon-edge .
+
+nodeid=$(./polygon-edge secrets init --data-dir test-chain- --num 4 | grep Node | head -n 1 | awk -F ' ' '{print $4}')
+
+./polygon-edge genesis --consensus ibft --ibft-validators-prefix-path test-chain- \
+  --bootnode /ip4/127.0.0.1/tcp/10001/p2p/"$nodeid" \
+  --premine 0x85da99c8a7c2c95964c8efd687e95e632fc533d6:1000000000000000000000 \
```

Comment:
If I'm correct, this doesn't do anything, since we are not sure that given address 0x85da99c8a7c2c95964c8efd687e95e632fc533d6 exists. Am I right? I would omit premine flag and rely on default premine balance (1M tokens/10^24 weis). WDYT?

```suggestion
```

Reply:
Currently, there is switch on `v3-parity`: `--premine-validators 1000000000000000000000` which can be used to premine some balance for all the validators. I am not sure if we gonna keep that switch after merging to develop

Reply:
@igorcrevar No need to do that because all the validators are going to get 1M of tokens if premine flag is not provided. 🙂

Reply:
FTR: The problem is that we don't have a fixed address for external testing, load testing tools and that, you know? so instead of update the address for testing on each run we rely on a fixed address that can be hardcoded in external tools.

Reply:
We had discussed this offline and it turns out that address which we are going to premine is going to be used by the load testing tools, so we want to make sure that we have premined appropriate address.